{"uid":"3e96781a2286ccce","name":"GetFranchiseList - page_size is zero","fullName":"GetFranchiseList - page_size is zero","historyId":"GetFranchiseList - page_size is zero","time":{"start":1761304288836,"stop":1761304289224,"duration":388},"status":"failed","statusMessage":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","statusTrace":"java.lang.AssertionError: The following asserts failed:\n\tMismatch in result size expected [0] but found [1]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\n\tat Listener.AllureLogs.executeSoftAssertAll(AllureLogs.java:182)\n\tat autochek_api_negative.UserService.UserServiceNegative.GetFranchiseList(UserServiceNegative.java:547)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setUp","time":{"start":1761304285995,"stop":1761304287616,"duration":1621},"description":"","status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","statusTrace":"java.lang.AssertionError: The following asserts failed:\n\tMismatch in result size expected [0] but found [1]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\n\tat Listener.AllureLogs.executeSoftAssertAll(AllureLogs.java:182)\n\tat autochek_api_negative.UserService.UserServiceNegative.GetFranchiseList(UserServiceNegative.java:547)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\n","steps":[{"name":"SoftAssert: Verify result size - Expected: '0', Actual: '1'","time":{"start":1761304289218,"stop":1761304289221,"duration":3},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"softAssert","value":"org.testng.asserts.SoftAssert@4a729612"},{"name":"actual","value":"1"},{"name":"expected","value":"0"},{"name":"fieldName","value":"result size"}],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Execute all soft assertions","time":{"start":1761304289222,"stop":1761304289223,"duration":1},"status":"failed","statusMessage":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","statusTrace":"java.lang.AssertionError: The following asserts failed:\n\tMismatch in result size expected [0] but found [1]\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\n\tat Listener.AllureLogs.executeSoftAssertAll(AllureLogs.java:182)\n\tat autochek_api_negative.UserService.UserServiceNegative.GetFranchiseList(UserServiceNegative.java:547)\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\n\tat org.testng.TestRunner.run(TestRunner.java:616)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\n\tat org.testng.TestNG.run(TestNG.java:1079)\n\tat org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)\n\tat org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)\n\tat org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)\n\tat org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)\n\tat org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)\n\tat org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)\n\tat org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)\n","steps":[],"attachments":[{"uid":"52cc4d89bc81ca98","name":"Soft Assertion Failures","source":"52cc4d89bc81ca98.txt","type":"text/plain","size":81}],"parameters":[{"name":"softAssert","value":"org.testng.asserts.SoftAssert@4a729612"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"fa342fa2a83276c6","name":"Request Log","source":"fa342fa2a83276c6.txt","type":"application/json","size":558},{"uid":"567560a22b0d7143","name":"Response Log","source":"567560a22b0d7143","type":"application/json","size":4029}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":3,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"package","value":"autochek_api_negative.UserService.UserServiceNegative"},{"name":"testClass","value":"autochek_api_negative.UserService.UserServiceNegative"},{"name":"testMethod","value":"GetFranchiseList"},{"name":"parentSuite","value":"API_Regression_Suite"},{"name":"suite","value":"Autochek_API_Automation"},{"name":"subSuite","value":"autochek_api_negative.UserService.UserServiceNegative"},{"name":"host","value":"runnervmwhb2z"},{"name":"thread","value":"2156@runnervmwhb2z.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Get Franchise List"},{"name":"feature","value":"Negative UserService"},{"name":"epic","value":"API Automation"},{"name":"severity","value":"high"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"history":{"statistic":{"failed":6,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":6},"items":[{"uid":"89103b6d3c7cfb34","status":"failed","statusDetails":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","time":{"start":1761303418943,"stop":1761303419407,"duration":464}},{"uid":"fb86d6d9da3946f2","status":"failed","statusDetails":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","time":{"start":1761302552836,"stop":1761302553250,"duration":414}},{"uid":"b5153283e9a4499c","status":"failed","statusDetails":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","time":{"start":1761301870171,"stop":1761301870580,"duration":409}},{"uid":"d243982a347df4e","status":"failed","statusDetails":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","time":{"start":1761301569647,"stop":1761301570216,"duration":569}},{"uid":"acee14ea29387e07","status":"failed","statusDetails":"The following asserts failed:\n\tMismatch in result size expected [0] but found [1]","time":{"start":1761300995257,"stop":1761300995695,"duration":438}}]},"tags":[]},"source":"3e96781a2286ccce.json","parameterValues":[]}